### PR TITLE
Added script, refactored services

### PR DESCRIPTION
**Pull request by DanilBut**

### CODE REVIEW

File: sandbox-backend/src/main/resources/db/changelog/changes/2021.03.05_dropNotNullConstraint_status_history.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+databaseChangeLog:
+  - changeSet:
+      id: add_dropNotNullConstraint_status_history
+      author: butenko
```
Comment (ikantsevich) [inline sandbox-backend/src/main/resources/db/changelog/changes/2021.03.05_dropNotNullConstraint_status_history.yaml:4]: please add precondition

**Review by ikantsevich — APPROVED** [review]

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/StatusHistory.java [hunk 2]
```diff
@@ -34,7 +34,7 @@
     private String changeNote;
 
     @CreationTimestamp
-    @Column(name = "STH_CREATED_AT", nullable = false)
+    @Column(name = "STH_CREATED_AT")
     private LocalDateTime createdAt;
```
Comment (ikantsevich) [inline sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/StatusHistory.java:38]: Why we should remove notnull?

**Review by ikantsevich — CHANGES_REQUESTED** [review]

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/StatusHistory.java (hunk 2 above)
Comment (DanilBut) [reply sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/StatusHistory.java]: returned it back

File: sandbox-backend/src/main/resources/db/changelog/changes/2021.03.05_dropNotNullConstraint_status_history.yaml (hunk 1 above)
Comment (DanilBut) [reply sandbox-backend/src/main/resources/db/changelog/changes/2021.03.05_dropNotNullConstraint_status_history.yaml:4]: corrected

Comment (ikantsevich) [reply sandbox-backend/src/main/resources/db/changelog/changes/2021.03.05_dropNotNullConstraint_status_history.yaml:4]: If we added default value for this column this changeset not need

**Review by ikantsevich — APPROVED** [review]

**Pull request merged**